iaac/azure/arm_templates/mysql_flexible_server/parameters.json

```diff
@@ -0,0 +1,88 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serverName": {
+            "value": "infra"
+        },
```

Comment:
I think that we can try to use the same ARM template for each DB
(+) easy of deployment
(-) possibility that infra DB will need more resources then other DBs